movie/src/components/PopupCard/BoxDisplayGenres/BoxDisplayGenres.jsx

```diff
@@ -0,0 +1,43 @@
+import { Box } from "@mui/material";
+import FilterPopup from "../FilterPopup/FilterPopup";
+import Collapse from "@mui/material/Collapse";
```

Comment:
`import { Box } from "@mui/material";` and `import Collapse from "@mui/material/Collapse";`
                                                                    👇 
                                   `import { Box, Collapse } from "@mui/material";`
Same goes for other places that import from  `Mui`